domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/note/dto/NoteDto.kt

```diff
@@ -19,10 +19,14 @@ data class NoteDto(
 
 enum class EntityType {
   GOAL,
+  INDUCTION,
+  REVIEW,
 }
 
 enum class NoteType(val entityType: EntityType) {
   GOAL(EntityType.GOAL),
   GOAL_ARCHIVAL(EntityType.GOAL),
   GOAL_COMPLETION(EntityType.GOAL),
+  REVIEW(EntityType.REVIEW),
```

Comment:
I've had to define the new NoteType for Reviews, and also added the NoteType for Inductions whilst I was here; along with the mapper changes below (to map the new enum values)